docs/considerations.rst

```diff
@@ -0,0 +1,214 @@
+##########################
+Considerations for ME-fMRI
+##########################
+Multi-echo fMRI acquisition sequences and analysis methods are rapidly maturing. Someone who has access
+to a multi-echo fMRI sequence should seriously consider using it. 
+
+The possible costs and benefits of multi-echo fMRI
+==================================================
+The following are a few points to consider when deciding whether or not to collect multi-echo data.
+
+Possible increase in TR
+-----------------------
+The one difference with multi-echo is a slight time cost.
+For multi-echo fMRI, the shortest echo time (TE) is essentially free since it is collected in the
+gap between the RF pulse and the single-echo acquisition.
+The second echo tends to roughly match the single-echo TE.
+Additional echoes require more time.
+For example, on a 3T MRI, if the T2* weighted TE is 30ms for single echo fMRI,
+a multi-echo sequence may have TEs of 15.4, 29.7, and 44.0ms.
+In this example, the extra 14ms of acquisition time per RF pulse is the cost of multi-echo fMRI.
+
+One way to think about this cost is in comparison to single-echo fMRI.
+If a multi-echo sequence has identical spatial resolution and acceleration as a single-echo sequence,
+then a rough rule of thumb is that the multi-echo sequence will have 10% fewer slices or 10% longer TR.
+Instead of compromising on slice coverage or TR, one can increase acceleration.
+If one increases acceleration, it is worth doing an empirical comparison to make sure there
+isn't a non-trivial loss in SNR or an increase of artifacts.
+
+Weighted Averaging may lead to an increase in SNR 
+-------------------------------------------------
+Multiple studies have shown that a
+weighted average of the echoes to optimize T2* weighting, sometimes called "optimally combined,"
+gives a reliable, modest boost in data quality. The optimal combination of echoes can currently be
+calculated in several software packages including AFNI, fMRIPrep, and tedana. In tedana, the weighted
+average can be calculated with `t2smap`_ If no other
+acquisition compromises are necessary to acquire multi-echo data, this boost is worthwhile. 
+
+Consider the life of the dataset
+--------------------------------
+If other
+compromises are necessary, consider the life of the data set. If data is being acquired for a discrete
+study that will be acquired, analyzed, and published in a year or two, it might not be worth making
+compromises to acquire multi-echo data. If a data set is expected to be used for future analyses in later
+years, it is likely that more powerful approaches to multi-echo denoising will sufficiently mature and add
+even more value to a data set.
+
+Other multi-echo denoising methods, such as MEICA, the predecessor to tedana, have shown the potential for
```

Comment:
I'm conflicted about this paragraph. I don't know that MEICA is _more powerful_, and tedana is (hopefully, eventually) where we'll see emerging methods added. I might rephrase around these lines ?